### PR TITLE
proposed fix for headless-gl issues

### PR DESCRIPTION
**Pull request by JonStargaryen**
Headless-gl requires a `[0]` suffix when getting the location of array uniforms (https://github.com/stackgl/headless-gl/issues/170). Without neither `uLightDirection` nor `uLightColor` get set.

See https://github.com/molstar/molrender/issues/3#issuecomment-1117615459.

### CODE REVIEW

File: src/mol-gl/webgl/uniform.ts [hunk 1]
```diff
@@ -42,6 +42,14 @@ export function getUniformType(gl: GLRenderingContext, kind: UniformKind) {
     }
 }
 
+export function isArrayUniform(kind: UniformKind) {
```
Comment (arose) [inline src/mol-gl/webgl/uniform.ts:45]: I think just `return kind.endsWith('[]');` is fine

**Review by arose — COMMENTED** [review]
Nice find, thank you @JonStargaryen.

**Comment by arose** [comment]
Also, please add a line to the changelog

File: src/mol-gl/webgl/uniform.ts (hunk 1 above)
Comment (JonStargaryen) [reply src/mol-gl/webgl/uniform.ts:45]: Sure, simplified with [cbc941f](https://github.com/molstar/molstar/pull/434/commits/cbc941f19388c222f501ebf5520c5a36adcedd6f).

Changelog is updated as well.

**Pull request merged**